Added opinion section

Capeur opened the pull request:
- Added Shadcn Carousel component
- Added an opinion section in homepage using the Carousel component

t-webber reviewed (changes requested):
Thanks for contributing, this looks really nice

I left a few comments, don't hesitate to ask if there is a point you don't understand or if you need more information/help!

Capeur commented:
- Removed tests from files en.ts and fr.ts
- Changed avis to opinion in .tsx
- Added scroll to the card only for the text so that the author always shows
- Changed button on pc so that they no longer exceed the navbar and removed them on phone (since you can just swipe)
- Added new feature of autoplay 

I tried my best to take your comments into account, but don't hesitate to tell me if you find my code lacking something. I am waiting for your answer about the issue in white dark theme about the cards to decide of a action to solve it.

Also I still don't quite understand why the tests don't go through

t-webber commented on src/app/[locale]/(static)/(client)/(home)/page.tsx:
The goal of the dark theme is for the page to be dark... and also not to become blind, the box beneath is white for the sole reason that you aren't allowed the change the background colour of a logo

t-webber commented on src/components/animations/MyCarousel.tsx:
good idea moving it to components

t-webber commented on src/components/ui/carousel.tsx:
It's safe to stage everything, the `.gitignore` will prevent you of staging bad things like the `node_modules`.

But `bun.lock`, the `migrations/` folder, all of this is safe to stage, and you are encouraged to do so

t-webber commented:
> Also I still don't quite understand why the tests don't go through

Here is a short notice, don't hesitate if you have question, or if I said something that is not true :sweat_smile: 

# Formatting

For the formatting, it likely means you haven't formatted your code. I encourage you to activate "formatOnSave" if you use vscode, with the "prettier" extension, in order to never have unformatted code.

The component from shadcn you added follows their formatting guidelines so it doesn't follow ours. To format it, either open it and save it to apply the prettier extension, or run `bun run fmt` to fix the issue.

# Build 

This is also a formatting error, but from eslint that expects you to separate the import order, for example:

```
import { redirect } from 'next/navigation';

// expected a separation here

import { auth } from '@/auth/auth';
import { Block } from '@/components/styles/blocks';
import { getBlog } from '@/db/blogs';
import { getLocaleLabels } from '@/db/labels';
import { Locale, LocalePostParams } from '@/locales/config';
import { nav } from '@/locales/routing';

// expected a separation here

import { QuillEditor } from './quill-editor';
import { UnValidate } from './unvalidate';
```

To fix this, I recommand you to install the eslint plugin if you use vscode, or similar. I believe it doesn't require any configuration (though my vscode knownledge is getting rusty).

To do in the command line, you can do `bun run lint --fix`
